docs_user/modules/proc_adopting-the-identity-service.adoc

```diff
@@ -2,15 +2,14 @@
 
 = Adopting the {identity_service}
 
+To adopt the {identity_service_first_ref}, you patch an existing `OpenStackControlPlane` custom resource (CR) where the {identity_service} is disabled. The patch starts the service with the configuration parameters that are provided by the {rhos_prev_long} ({OpenStackShort}) environment.
+
 .Prerequisites
 
-* Previous Adoption steps completed. Notably,
- the xref:migrating-databases-to-mariadb-instances_migrating-databases[Migrating databases to MariaDB instances]
-must already be imported into the control plane MariaDB.
-* Ensure that you copy the fernet keys. Create the `keystone` secret, containing fernet keys:
+* Ensure that you copy the fernet keys. Create the keystone secret, containing fernet keys:
```

Comment:
@xek Is the prerequisite 2 separate steps? Copy the fernet keys, then create the keystone secret containing the fernet keys?

Reply:
Maybe a better wording would be:

Create the keystone secret, containing the fernet keys copied over from the source environment.